quickstep/src/com/android/launcher3/statehandlers/DepthController.java

```diff
@@ -134,8 +139,10 @@ public void setStateWithAnimation(LauncherState toState, StateAnimationConfig co
 
     @Override
     public void applyDepthAndBlur() {
-        ensureDependencies();
-        super.applyDepthAndBlur();
+        if(app.lawnchair.LawnchairApp.isAtleastT()){
```

Comment:
```suggestion
        if (app.lawnchair.LawnchairApp.isAtleastT()) {
```

Reply:
The same issue is happening multiple times in this PR.

Reply:
It's intended , cause if not this error occurs  : Abort message: 'JNI DETECTED ERROR IN APPLICATION: JNI CallVoidMethodV called with pending exception java.lang.NoSuchMethodError: No virtual method setWallpaperZoomOut(Landroid/os/IBinder;F)V in class Landroid/app/WallpaperManager; or its super classes (declaration of 'android.app.WallpaperManager' appears in /system/framework/framework.jar)
                                                                                                      at void com.android.quickstep.util.BaseDepthController.applyDepthAndBlur() (BaseDepthController.java:114)
                                                                                                      at void com.android.launcher3.statehandlers.DepthController.applyDepthAndBlur() (DepthController.java:143)
                                                                                                      at void com.android.launcher3.statehandlers.DepthController$1.onViewAttachedToWindow(android.view.View) (DepthController.java:85)
                                                                                                      at void com.android.launcher3.statehandlers.DepthController.ensureDependencies() (DepthController.java:98)
                                                                                                      at void com.android.launcher3.statehandlers.DepthController.applyDepthAndBlur() (DepthController.java:142)
                                                                                                      at void com.android.quickstep.util.BaseDepthController.setDepth(float) (BaseDepthController.java:159)

Reply:
Interesting! Okay, nevermind then. 👍🏼